src/gui/SetupDialog.cpp

```diff
@@ -124,7 +124,8 @@ SetupDialog::SetupDialog( ConfigTabs _tab_to_open ) :
 #endif
 	m_backgroundArtwork( QDir::toNativeSeparators( ConfigManager::inst()->backgroundArtwork() ) ),
 	m_smoothScroll( ConfigManager::inst()->value( "ui", "smoothscroll" ).toInt() ),
-	m_enableAutoSave( ConfigManager::inst()->value(	"ui", "enableautosave" ).toInt() ),
+	m_disableAutoSave( !ConfigManager::inst()->value( "ui", "disableautosave" ).toInt() ),
+	m_disableRunningAutoSave( !ConfigManager::inst()->value( "ui", "disablerunningautosave" ).toInt() ),
```

Comment:
Double negations should be avoided; "enable" should be preferred over "disable". This code does not make sense, because it looks like:
```
m_disableAutoSave = !disableautosave
m_disableRunningAutoSave = !disablerunningautosave
```